docs/classes/lsp7-digital-asset.md

```diff
@@ -167,7 +167,7 @@ Deployment Complete
 ```
 
 [constructor parameters]: ../../../../../standards/smart-contracts/lsp7-digital-asset#constructor
-[contract deployment options]: ../deployment/contract-deployment-options
+[contract deployment options]: ../deployment/digital-asset.md
```

Comment:
Maybe this could also refer to the anchor link  `../deployment/digital-asset/#deployment-configuration`, like it is done in 'lsp8-identifiable-digital-asset.md'?